frontend/data/locale/en-US.ini

```
@@ -630,15 +630,20 @@ RenameProfile.Title="Rename Profile"
 Basic.Main.MixerRename.Title="Rename Audio Source"
 Basic.Main.MixerRename.Text="Please enter the name of the audio source"
 
-
 # preview window disabled
 Basic.Main.PreviewDisabled="Preview is currently disabled"
 
 # add source dialog
-Basic.SourceSelect="Create/Select Source"
-Basic.SourceSelect.CreateNew="Create new"
-Basic.SourceSelect.AddExisting="Add Existing"
+Basic.SourceSelect="Add Source"
+Basic.SourceSelect.SelectType="Source Type"
+Basic.SourceSelect.Recent="Recently Added"
+Basic.SourceSelect.NewSource="Create A New Source"
+Basic.SourceSelect.Existing="Add An Existing Source"
```

If using Title Case, do not capitalize articles:
```suggestion
Basic.SourceSelect.NewSource="Create a New Source"
Basic.SourceSelect.Existing="Add an Existing Source"
```